src/dispatch/static/dispatch/src/case/Page.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <PageHeader
+      :case-name="caseDetails.name"
+      :case-visibility="caseDetails.visibility"
+      :case-status="caseDetails.status"
+      :is-drawer-open="isDrawerOpen"
+      @toggle-drawer="toggleDrawer"
+    />
+    <CaseAttributesDrawer v-model="caseDetails" :open="isDrawerOpen" />
+    <VDivider />
+    <div class="container mx-auto px-4">
+      <Tiptap :title="true" v-model="caseDetails.title" class="pl-8 pb-6 pt-6" />
```

Comment:
Is this component named correctly? It's not clear what it is. Perhaps something like dynamic input? The component itself seems to be a combination of things depending it's its prop. Can it be broken up / abstracted?

Reply:
I originally had it broken up but it felt like it added lots of boilerplate. We can revisit, I'm not 100% sure the best way to abstract it at the moment, ideally we can control sizes and headings/styles based on props, I think. Which is sort of what we do right now.

I did rename it.